radix-engine-monkey-tests/src/lib.rs

```diff
@@ -417,7 +416,7 @@ impl<T: TxnFuzzer> FuzzTest<T> {
                         TWO_RESOURCE_POOL_BLUEPRINT_IDENT,
                         TWO_RESOURCE_POOL_INSTANTIATE_IDENT,
                         TwoResourcePoolInstantiateManifestInput {
-                            resource_addresses: (pool_resource1, pool_resource2),
+                            resource_addresses: (pool_resource1.into(), pool_resource2.into()),
```

Comment:
Side note: Will be nice when we can tweak `call_function` to either take a `ManifestTwoResourcePoolInstantiateInput` or a `TwoResourcePoolInstantiateInput` ^_^

Reply:
Or even when we can rename `call_function` to `call_function_manual` and have it be `call_function(TwoResourcePoolInstantiateInput { .. })` or `call_function(ManifestTwoResourcePoolInstantiateInput { .. })`, via the `IsFunctionInput` trait we discussed in a previous PR ✨